hawc/services/epa/dsstox.py

```diff
@@ -29,16 +30,15 @@ def create_from_dtxsid(cls, dtxsid: str) -> "DssSubstance":
         """
         if not re.compile(RE_DTXSID).fullmatch(dtxsid):
             raise ValueError(f"Invalid DTXSID: {dtxsid}")
-
         response = requests.get(
-            f"https://comptox.epa.gov/dashboard-api/ccdapp2/chemical-detail/search/by-dsstoxsid?id={dtxsid}"
+            f"https://api-ccte.epa.gov/chemical/detail/search/by-dtxsid/{dtxsid}",
+            headers={"x-api-key": settings.CCTE_API_KEY, "Content-Type": "application/json"},
         )
         response_dict = response.json()
-        if response_dict == [] or response_dict["dsstoxSubstanceId"] != dtxsid:
+        if response_dict.get("dtxsid") != dtxsid:
             raise ValueError(f"{dtxsid} not found in DSSTox lookup")
 
-        response_dict["dtxsid"] = response_dict.pop("dsstoxSubstanceId")
-        response_dict.pop("presenceInLists")
+        response_dict["dtxsid"] = response_dict.pop("dtxsid")
```

Comment:
delete this line? it looks like it just pops and puts back on the same value in the same place